### PR TITLE
Replace variable name from 'query' to 'path' where query was defined as a full path(#259)

### LINKED ISSUE #259: Change naming of API path variable from `query` to `path`
In this example, we use `let query` to define the API path. As per this comment: https://github.com/meilisearch/meilisearch-swift/pull/248#discussion_r802883984

We should change the wording from `query` to `path`  everywhere `query` is used to define a path to an API route.

Example: 
https://github.com/meilisearch/meilisearch-swift/blob/f322911570ae4d7836fd6f1bce49d1dec6ba12b2/Sources/MeiliSearch/Documents.swift#L21

Should become
`let path`

### PR DESCRIPTION
**Pull request by Dishant10**
# Pull Request

## Related issue
Fixes #259 

## What does this PR do?
Fixes the issue where a variable name 'query' was set to a full path value. As the issue mentioned the variable name 'path' would be more suitable in this situation. So this PR changes all the variable query whose values were actually a path.
- ...

## PR checklist
Please check if your PR fulfills the following requirements:
- [ • ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ • ] Have you read the contributing guidelines?
- [ •  ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: Sources/MeiliSearch/Documents.swift [hunk 1]
```diff
@@ -14,22 +14,22 @@ struct Documents {
     self.request = request
   }
 
-  // MARK: Query
+  // MARK: Path
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:17]: I think we could remove this line entirely 
```suggestion
```

File: Sources/MeiliSearch/Documents.swift [hunk 2]
```diff
@@ -14,22 +14,22 @@ struct Documents {
...
     _ completion: @escaping (Result<T, Swift.Error>) -> Void)
   where T: Codable, T: Equatable {
-    var query: String = "/indexes/\(uid)/documents/\(identifier)"
+    var path: String = "/indexes/\(uid)/documents/\(identifier)"
 
     if fields != nil {
-      let fieldsQuery = "?fields=\(fields?.joined(separator: ",") ?? "")"
-      query.append(fieldsQuery)
+      let fieldsPath = "?fields=\(fields?.joined(separator: ",") ?? "")"
+      path.append(fieldsPath)
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:29]: In this case the "query" name is correct :)
```suggestion
      let fieldsQuery = "?fields=\(fields?.joined(separator: ",") ?? "")"
      path.append(fieldsQuery)
```

**Review by brunoocasali — APPROVED** [review]
Thanks a lot for your contribution to Meilisearch @Dishant10 ❤️

**Comment by brunoocasali** [comment]
@Dishant10 apparently, your code does not matches our linter rules, can you check that?

File: Sources/MeiliSearch/Documents.swift [hunk 3]
```diff
@@ -14,22 +14,21 @@ struct Documents {
     self.request = request
   }
 
-  // MARK: Query
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:17]:
```suggestion
```

File: Sources/MeiliSearch/Documents.swift [hunk 4]
```diff
@@ -73,12 +72,12 @@ struct Documents {
     _ primaryKey: String? = nil,
     _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) {
 
-    var query: String = "/indexes/\(uid)/documents"
+    var path: String = "/indexes/\(uid)/documents"
     if let primaryKey: String = primaryKey {
-      query += "?primaryKey=\(primaryKey)"
+        path += "?primaryKey=\(primaryKey)"
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:77]:
```suggestion
      path += "?primaryKey=\(primaryKey)"
```

File: Sources/MeiliSearch/Documents.swift [hunk 5]
```diff
@@ -94,9 +93,9 @@ struct Documents {
     _ encoder: JSONEncoder? = nil,
     _ primaryKey: String? =  nil,
     _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) where T: Encodable {
-    var query: String = "/indexes/\(uid)/documents"
+    var path: String = "/indexes/\(uid)/documents"
     if let primaryKey: String = primaryKey {
-      query += "?primaryKey=\(primaryKey)"
+        path += "?primaryKey=\(primaryKey)"
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:98]:
```suggestion
      path += "?primaryKey=\(primaryKey)"
```

File: Sources/MeiliSearch/Documents.swift [hunk 6]
```diff
@@ -124,12 +123,12 @@ struct Documents {
     _ primaryKey: String? = nil,
     _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) {
 
-    var query: String = "/indexes/\(uid)/documents"
+    var path: String = "/indexes/\(uid)/documents"
     if let primaryKey: String = primaryKey {
-      query += "?primaryKey=\(primaryKey)"
+        path += "?primaryKey=\(primaryKey)"
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:128]:
```suggestion
      path += "?primaryKey=\(primaryKey)"
```

File: Sources/MeiliSearch/Documents.swift [hunk 7]
```diff
@@ -146,9 +145,9 @@ struct Documents {
     _ primaryKey: String? =  nil,
     _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) where T: Encodable {
 
-    var query: String = "/indexes/\(uid)/documents"
+    var path: String = "/indexes/\(uid)/documents"
     if let primaryKey: String = primaryKey {
-      query += "?primaryKey=\(primaryKey)"
+        path += "?primaryKey=\(primaryKey)"
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Documents.swift:150]:
```suggestion
      path += "?primaryKey=\(primaryKey)"
```

**Comment by Dishant10** [comment]
Sure! I'll take a look at that.

**Review by brunoocasali — APPROVED** [review]
Thanks a lot for your contribution to Meilisearch ❤️ 

bors merge

**Comment by Dishant10** [comment]
I read the CONTRIBUTING.md for the linter issues. Should I run this command? 
swift run swiftlint --fix
I don't have much knowledge about linter/linting.

**Comment by brunoocasali** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**